docs/upgrading.rst

```diff
@@ -11,6 +11,11 @@ upgrade notes if your version is older than 3.4.2.
 any version to 4.8.x
 --------------------
 
+Use of SHA1 on Enterprise Linux 9 and derivatives
```

Comment:
should this header more clearly mention Red Hat?

Reply:
It probably should; if 'derivatives' are mentioned explicitly, then 'EL9' doesn't exist, it's 'RHEL9'.

Reply:
adjusted